pos_accented_search/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "pos_accented_search",
+
+    'summary': """
+        Better product search in POS. Accented characters like é are normalized.""",
+
+    'description': """
+        This add-on make pos product search insensitive to accented characters in the product
+        name. For instance, café will match both cafe and café.  
+    """,
+
+    'author': "Le Nid",
```

Comment:
this also needs to contain the OCA: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#modules (you should read and follow all of this, otherwise review becomes tedious. I'm going to ignore everything mentioned in the conventions from now on)

Reply:
Will do that thank you.

Reply:
Don't hesitate to reach me for assistance.

Reply:
Thank you, I'll give it a try and call for help if needed!